pages/treeherder.py

```diff
@@ -25,6 +25,12 @@ class TreeherderPage(Base):
     _filter_panel_reset_locator = (By.CSS_SELECTOR, '.pull-right span:nth-child(3)')
     _filter_panel_testfailed_failures_locator = (By.ID, 'testfailed')
     _mozilla_central_repo_locator = (By.CSS_SELECTOR, '#th-global-navbar-top a[href*="mozilla-central"]')
+    _nav_filter_coalesced_locator = (By.CSS_SELECTOR, '.btn-ltblue-filter-chicklet')
+    _nav_filter_failures_locator = (By.CSS_SELECTOR, '.btn-red-filter-chicklet')
+    _nav_filter_inprogress_locator = (By.CSS_SELECTOR, '.btn-dkgray-filter-chicklet')
+    _nav_filter_retry_locator = (By.CSS_SELECTOR, '.btn-dkblue-filter-chicklet')
+    _nav_filter_successes_locator = (By.CSS_SELECTOR, '.btn-green-filter-chicklet')
+    _nav_filter_usercancel_locator = (By.CSS_SELECTOR, '.btn-pink-filter-chicklet')
```

Comment:
We should locate these elements by the `title` rather than the `class`, as the class relates to the style of the element and not the functionality. If the color of any of these changed, it would cause a functional test to fail.